Paws4Me/Paws4Me/Controllers/Logic/PetSingleDetailsViewController.swift

```diff
@@ -8,52 +8,45 @@
 import UIKit
 
 class PetSingleDetailsViewController: UIViewController {
+    // MARK: - IBOutlets
     @IBOutlet weak private var petImageView: UIImageView!
     @IBOutlet weak private var petName: UILabel!
     @IBOutlet weak private var petAge: UILabel!
     @IBOutlet weak private var petGender: UILabel!
     @IBOutlet weak private var petBreedName: UILabel!
+
+    // MARK: - Vars/Lets
     var singlePet: AdoptPet?
```

Comment:
These can be private

Reply:
Thank you for the suggestion however, I need these specific variables to be public as I am passing the values of each of these variables from my allPetViewContorller to the PetSingleDetailsViewController through a segue

Reply:
Rather add some setters functions 😄

Reply:
Will do so 👍🏻